src/App.jsx

```diff
@@ -14,33 +19,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
+function sorteGoods (goods, sortedField, reversed) {
+  goods.sort((a, b) => {
+    switch (sortedField) {
+      case SORT_BY_ALPHABET:
+        return a.localeCompare(b);
+      case SORT_BY_LENGTH:
+        return a.length - b.length;
+      default:
+        return 0;
+    }
+  })
+
+  if (reversed) {
+    goods.reverse();
+  }
+
+  return goods;
+}
+export const App = () => {
+  const [sortedField,setSortedField] = useState('');
+  const [reversed, setReversed] = useState(false);
+
+  let goods = sorteGoods([...goodsFromServer], sortedField, reversed);
+
+  function reset () {
```

Comment:
The function name 'reset' should start with a verb to better reflect its action-oriented purpose. Consider renaming it to 'resetSorting' or 'resetGoods'. This aligns with the checklist item: 'Method's name should start with a verb'.